dev.overlayed.Overlayed.yaml

```diff
@@ -0,0 +1,96 @@
+    sources:
+      - type: archive
+        url: https://dotat.at/prog/unifdef/unifdef-2.12.tar.xz
+        sha256: 43ce0f02ecdcdc723b2475575563ddb192e988c886d368260bc0a63aee3ac400
+    cleanup:
+      - "*"
+  # NOTE: native deps
+  # https://github.com/tauri-apps/tauri-docs/blob/v1/docs/guides/building/cross-platform.md?plain=1#L113
+  # - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - name: webkit2gtk-4.0
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.44.0.tar.xz
+        sha256: c66530e41ba59b1edba4ee89ef20b2188e273bed0497e95084729e3cfbe30c87
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+    buildsystem: cmake-ninja
+    config-opts:
+      # NOTE: not sure flat webkit2gtk-4.0 flags tauri requires
+      - -DPORT=GTK
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DENABLE_DOCUMENTATION=OFF
+      - -DENABLE_MINIBROWSER=OFF
+      - -DENABLE_WEBDRIVER=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DUSE_LIBBACKTRACE=OFF
+      - -DUSE_GTK4=OFF
+      - -DUSE_SOUP2=ON
+      - -DENABLE_BUBBLEWRAP_SANDBOX=OFF
+  - name: overlayed
+    buildsystem: simple
+    build-commands:
+      - chmod +x Overlayed.AppImage
+      - ./Overlayed.AppImage --appimage-extract
+
+      - |
+        desktop-file-edit \
+          --set-name="Overlayed" \
+          --set-comment="Overlayed's desktop application" \
+          --set-icon="$FLATPAK_ID" \
+          --set-key="StartupNotify" --set-value="true" \
+          --set-key="Categories" --set-value="Game;" \
+          --set-key="Keywords" --set-value="discord;overlay;" \
+          --set-key="StartupWMClass" --set-value="Overlayed" \
+          squashfs-root/usr/share/applications/overlayed.desktop
+
+      - mkdir -p /app/{bin,share}
+      - cp -r squashfs-root/usr/share/icons /app/share/icons
+      - install -Dm755 squashfs-root/usr/bin/overlayed /app/bin/overlayed
+      - install -Dm644 squashfs-root/usr/share/applications/overlayed.desktop /app/share/applications/dev.overlayed.Overlayed.desktop
+      - install -Dm644 dev.overlayed.Overlayed.metainfo.xml /app/share/metainfo/dev.overlayed.Overlayed.metainfo.xml
+
+    sources:
+      - type: file
+        url: https://raw.githubusercontent.com/overlayeddev/overlayed/14a6d2c8b022f7d44de4a57350e8f30ff35ef7dc/distribution/dev.overlayed.Overlayed.metainfo.xml
+        sha256: 35773ae27a47753852aac2ace795675cef222d5d9681e5cdd93d92b6d16831c5
+
+      # NOTE: skipping building from source as there is no direct pnpm support as of yet
+      # https://github.com/flathub/flathub/pull/3876
+      # https://github.com/flatpak/flatpak-builder-tools/issues/383
+      - type: file
+        url: https://github.com/overlayeddev/overlayed/releases/download/v0.5.0/overlayed_0.5.0_amd64.AppImage
```

Comment:
you mean the appimage doesn't come with webkitgtk? What the point of it if it can't work on many system?